extensions/wikia/Wall/WallHooksHelper.class.php

```diff
@@ -2339,8 +2339,11 @@ public static function onArticleCommentGetSquidURLs( $title, &$urls ) {
 	 */
 	private static function shouldBoldenFollowedLink( User $user, Title $title, WallMessage $wm ) {
 		$isWatchlist = $title->isSpecial( 'Watchlist' );
```

Comment:
Its a little weird that we're using "not Watchlist" (line 2340) as a test for "is RecentChanges".  I know this code has had a lot of updates, but it might be better to test for what we're looking for, e.g.:
```php
$isRecentChanges = $title->isSpecial( 'RecentChanges' );
...
&& ( $isRecentChanges ... )
```